src/avgScrabbleScore.js

```diff
@@ -8,11 +8,13 @@ let scrabbleScore = require('./scrabbleScore');
  */
 function avgScrabbleScore(words) {
   // This is your job. :)
+  return scrabbleScore(words) / words.length;
```

Comment:
`scrabbleScore` is meant to take a string as input and `avgScrabbleScore` is meant to take an array, so this couldn't work.

If `words` is a string then `words.length` is _not_ the average score per word, but rather the average score per letter.